help/sources/tutorials/ui/create/customer-success/zendesk.md

```diff
@@ -0,0 +1,141 @@
+
+## Connect your Zendesk account
+
+In the Platform UI, select **[!UICONTROL Sources]** from the left navigation bar to access the [!UICONTROL Sources] workspace. The [!UICONTROL Catalog] screen displays a variety of sources with which you can create an account.
+
+You can select the appropriate category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the *Customer Success* category, select *Zendesk*, and then select **[!UICONTROL Add data]**.
+
+![catalog](../../../../images/tutorials/create/zendesk/catalog.png)
+
+The **[!UICONTROL Connect Zendesk account]** page appears. On this page, you can either use new credentials or existing credentials.
+
+### Existing account
+
+To use an existing account, select the *Zendesk* account you want to create a new dataflow with, then select **[!UICONTROL Next]** to proceed.
+
+![existing](../../../../images/tutorials/create/zendesk/authentication-existing-account.png)
+
+### New account
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![new](../../../../images/tutorials/create/zendesk/authentication-new-account.png)
+
+## Next steps
+
+By following this tutorial, you have established a connection to your *Zendesk* account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](https://experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/dataflow/crm.html).
+
+## Additional resources
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source; and Zendesk profiles are being ingested, follow the steps below:
+
+1. In the Platform UI, select **[!UICONTROL Datasets]** from the left navigation bar to access the [!UICONTROL Datasets] workspace. The [!UICONTROL Dataset Activity] screen displays the details of executions.
+![Activity page](../../../../images/tutorials/create/zendesk/dataset-activity.png?lang=en)
+![Dataflow page](../../../../images/tutorials/create/zendesk/dataflow-monitoring.png?lang=en)
+
+1. Next, select the **[!UICONTROL Data governance]** tab, you will see the schema details.
+![Zendesk schema](../../../../images/tutorials/create/zendesk/dataset-governance.png?lang=en)
+
+1. Finally, select the **[!UICONTROL Preview dataset]** button to display the data that was ingested.
+![Zendesk dataset](../../../../images/tutorials/create/zendesk/preview-dataset.png?lang=en)
+
+### Zendesk schema
+
+* A list of supported mappings that are to be set up for the [!DNL Zendesk Search API > Export Search Results](https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/#export-search-results) is given below.
```

Comment:
Can we rephrase this to something like: "The table below outlines a list of supported mappings that must be set up for the [!DNL Zendesk Search API > Export Search Results](https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/#export-search-results)."